cat-launcher/src/pages/BackupsPage/columns.tsx

```diff
@@ -4,11 +4,16 @@ import { ArrowUpDown } from "lucide-react";
 import { Badge } from "@/components/ui/badge";
 import { Button } from "@/components/ui/button";
 import { BackupEntry } from "@/generated-types/BackupEntry";
+import { ManualBackupEntry } from "@/generated-types/ManualBackupEntry";
+
+type CombinedBackup =
```

Comment:
Extract this type into another file and export it there. Then use this type everywhere type with backup is required.